### DISCUSSION TITLE
Fingerbot Moes

### DISCUSSION THREAD

**Discussion by giostefax89** [custom-components/ble_monitor#1127] [Q&A]
Good morning everyone.

I bought some moes fingerbots on aliexpress
 https://it.aliexpress.com/item/1005003134582092.html?spm=a2g0o.order_list.order_list_main.280.71e03696wEtTfv&gatewayAdapt=glo2ita

is it possible to integrate them via passive bluetooth?

Thank you

**Comment by Ernst79** [comment]
Please check the documentation how to collect some data, such that we can find out if that is possible. There is a special page with instructions.
